cmd/a2o-migrate/a2o/commit.go

```diff
@@ -0,0 +1,34 @@
+package a2o
+
+import (
+	"path/filepath"
+
+	"github.com/sirupsen/logrus"
+)
+
+// Commit finalises the migration by deleting leftover data.
+func Commit() error {
+	logrus.Info("commiting changes")
+
+	// remove aufs layer data
+	err := removeDirIfExists(aufsRoot())
+	if err != nil {
+		return err
+	}
+
+	// remove images
+	aufsImageDir := filepath.Join(StorageRoot, "image", "aufs")
+	err = removeDirIfExists(aufsImageDir)
+	if err != nil {
+		return err
+	}
+
+	// remove hostapps
+	hostappsDir := filepath.Join(StorageRoot, "hostapps")
```

Comment:
This seems to be very balenaOS specific.
Should we consider parameterizing this from outside? For instance, our update scripts in the OS could specify extra directories to remove on commit.

Reply:
good catch :heavy_check_mark: